Node Status Monitor/NodeJS/node_monitor.js

```diff
@@ -0,0 +1,197 @@
+/* imports*/
+import fetch from 'node-fetch';
+import fs from 'fs';
+import log from './utils/logger.js';
+import { get_token } from './api/auth.js';
+import config from './config/config.js';
+
+
+/* 
+variables:
+        Key                             Description
+    Only_changed_data :  if true get only changed node status, all node with status
+    
+functions:
+        Function Name                   Description
+    fetch_Data            :     function to fetch Node Details
+    process_node_status   :     function to process Node Details
+    write_data            :     function to write Node Details
+    read_old_data         :     function to read old Node Details
+    compare_data          :     function to compare  Node Details 
+    get_token             :     function to get Access token 
+    
+Files:
```

Comment:
The files are from config so the filenames may not be correct as hardcoded.

Reply:
removed hardcoded name